File: Examples/src/main/java/com/aspose/words/examples/quickstart/AppendDocuments.java
package com.aspose.words.examples.quickstart;

import com.aspose.words.Document;
import com.aspose.words.ImportFormatMode;
import com.aspose.words.examples.Utils;

public class AppendDocuments {
    public static void main(String[] args) throws Exception {
        //ExStart:
        // The path to the documents directory.
        String dataDir = Utils.getDataDir(AppendDocuments.class);

        // Load the destination and source documents from disk.
        Document dstDoc = new Document(dataDir + "TestFile.Destination.doc");
        Document srcDoc = new Document(dataDir + "TestFile.Source.doc");
        // Append the source document to the destination document while keeping the original formatting of the source document.
        dstDoc.appendDocument(srcDoc, ImportFormatMode.KEEP_SOURCE_FORMATTING);
        dstDoc.save(dataDir + "TestFile Out.docx");
        //ExEnd:

        System.out.println("Documents appended successfully.");
    }
}

File: Plugins/Aspose_Words_Java_for_Jython/asposewords/quickstart/AppendDocuments.py
from asposewords import Settings
from com.aspose.words import Document
from com.aspose.words import ImportFormatMode

class AppendDocuments:

    def __init__(self):
        dataDir = Settings.dataDir + 'quickstart/'
        
        dstDoc = Document(dataDir + 'TestFile.Destination.doc')
        srcDoc = Document(dataDir + 'TestFile.Source.doc')

        dstDoc.appendDocument(srcDoc,ImportFormatMode.KEEP_SOURCE_FORMATTING)
        
        dstDoc.save(dataDir + 'AppendDocuments.doc')
        
        print ("Documents appended successfully.")
        
if __name__ == '__main__':        
    AppendDocuments()